src/chart/generateCategoricalChart.tsx

```diff
@@ -1471,8 +1543,11 @@ export const generateCategoricalChart = ({
       const mouse = this.getMouseInfo(e);
       const nextState: CategoricalChartState = mouse ? { ...mouse, isTooltipActive: true } : { isTooltipActive: false };
 
-      this.setState(nextState);
-      this.triggerSyncEvent(nextState);
+      // If the dev set a defaultIndex, don't set `isTooltipActive` to false
+      if (nextState.isTooltipActive || typeof this.props.defaultIndex !== 'number') {
```

Comment:
don't think we need the number check here

Reply:
So, I figured that if someone wanted the tooltip to be visible from the beginning, that they'd always want it to be visible. 
So, if `defaultIndex` was set, I don't want `isTooltipActive` to ever be false.
So, in the 2 `*MouseLeave` methods, I'd only allow `isTooltipActive=false` if `defaultIndex` _wasn't_ set. 

Using `typeof defaultIndex === "number"` was just how I was checking if it had been set.

At this point, I think I was just adding too much complexity. I think the decision to make the tooltips always visible should be separate from where the tooltip should appear by default. I've backed out these checks, to reduce the scope of this PR.